algorithm/sorting-algo-python/bogo_sort.py

```diff
@@ -0,0 +1,22 @@
+from itertools import permutations

```

Comment:
Same for this one. We don'tt have any libraries here in this repo. Maybe you should try doing everything raw/ manually here.